lte/gateway/configs/sessiond.yml

```diff
@@ -52,7 +52,9 @@ sessions_table: sessiond:sessions
 
 # Set to true if converged access set message support is required or 5g access
 # support is required
-enable5g_features: false
+# Commenting local config parameter for enable5g_features to give precedence
+# to Orc8r streamed config parameter in gateway.mcofnig file
```

Comment:
same typo

Reply:
@ulaskozat In haste, this typo was introduced. Could you please let me know if it is ok to fix this typo in any upcoming PR?

Reply:
yes

Reply:
@ulaskozat The typo has been corrected in PR #11101